otel-extension/src/main/java/io/scalac/mesmer/otelextension/MesmerAkkaStreamInstrumentationModule.java

```diff
@@ -26,26 +26,27 @@ public MesmerAkkaStreamInstrumentationModule() {
     @Override
     public List<TypeInstrumentation> typeInstrumentations() {
 
-        TypeInstrumentation connectionConst = new TypeInstrumentation() {
-            @Override
-            public ElementMatcher<TypeDescription> typeMatcher() {
-                return ElementMatchers.named("akka.stream.impl.fusing.GraphInterpreter$Connection");
-            }
-
-            @Override
-            public void transform(TypeTransformer transformer) {
-                transformer.applyAdviceToMethod(isConstructor(), "akka.stream.ConnectionConstructorAdvice");
-            }
-        };
-
-
-        List<TypeInstrumentation> streamInstrumentations = new ArrayList<>();
-
-        streamInstrumentations.add(connectionConst);
-
-        streamInstrumentations.addAll(AkkaStreamAgent.agent().asOtelTypeInstrumentations());
-
-        return streamInstrumentations;
+//        TypeInstrumentation connectionConst = new TypeInstrumentation() {
+//            @Override
+//            public ElementMatcher<TypeDescription> typeMatcher() {
+//                return ElementMatchers.named("akka.stream.impl.fusing.GraphInterpreter$Connection");
+//            }
+//
+//            @Override
+//            public void transform(TypeTransformer transformer) {
+//                transformer.applyAdviceToMethod(isConstructor(), "akka.stream.ConnectionConstructorAdvice");
+//            }
+//        };
+
+
+//        List<TypeInstrumentation> streamInstrumentations = new ArrayList<>();
+//
+//        streamInstrumentations.add(connectionConst);
+//
+//        streamInstrumentations.addAll(AkkaStreamAgent.agent().asOtelTypeInstrumentations());
+//
+//        return streamInstrumentations;
+        return AkkaStreamAgent.agent().asOtelTypeInstrumentations();
```

Comment:
Note for the future: please never leave commented out code

```suggestion
        return AkkaStreamAgent.agent().asOtelTypeInstrumentations();
```

Reply:
I'm not sure if we even need provided->compile in agent sbt config as well. [It means](https://www.scala-sbt.org/1.x/docs/Multi-Project.html#Per-configuration+classpath+dependencies) that "provided configuration in agent depends on compile configuration in core". Why would such sophisticated configuration be needed in agent that is meant only to be a fat jar added as a javaagent (not as a provided-scope dependency)? 

Anyways when we merge everything in otel-extension It will work in both variants.